src/hooks/useLoadClientScript.ts

```diff
@@ -80,7 +80,7 @@ export function useLoadClientScript(props: NostoScriptProps) {
         document.body.appendChild(script)
       }
     }
-  }, [clientScriptLoaded, shopifyMarkets])
```

Comment:
clientScriptLoaded changes shouldn't retrigger the execution of the useEffect body